Library/Homebrew/os/mac/mach.rb

```diff
@@ -202,3 +199,7 @@ def dylib_id
     mach_metadata.dylib_id
   end
 end
+
+class Pathname
+  include(MachO)
+end
```

Comment:
I think it's better this lives in `Library/Homebrew/extend/pathname.rb` or `Library/Homebrew/extend/pathname` somewhere.